app/index.js

```diff
@@ -50,30 +50,70 @@ var BreiAppGenerator = yeoman.generators.Base.extend({
 	},
 
 	writing: {
-		app: function () {
+		folders: function () {
 			this.dest.mkdir('app');
+
+			// Assembled HTML
 			this.dest.mkdir('app/modules');
+			// Compiled CSS
 			this.dest.mkdir('app/css');
+			// Your scripts
 			this.dest.mkdir('app/js');
-			this.dest.mkdir('app/img');
+			this.dest.mkdir('app/js/plugins');
+			this.dest.mkdir('app/js/modules');
+			this.dest.mkdir('app/js/lib');
```

Comment:
The new js layout is actually because thats how I saw projects ending up one way or another. 

`modules` being where your scripts for individual modules were kept.
`plugins` being where scripts relating to plugins (like bxslider) was kept. 
`lib` being where you had scripts that affect multiple parts of the project.

I didnt think about bower installing anything directly in them though and I think thats something we can definitely do.